_plugins/after_init.rb

```diff
@@ -0,0 +1,16 @@
+# Copy the specified documentation repos into the jekyll-tracked "doc" directory.
+require 'fileutils'
+
+Jekyll::Hooks.register :site, :after_init do |site|
+    site.config['documentation_repos'].each do |repo_dir|
+        origin = File.join(site.source, "_remotes", repo_dir)
+        dest = File.join(site.source, "doc") 
+        to_delete = File.join(site.source, repo_dir) 
+        if File.directory?(to_delete)
+        FileUtils.rm_r(to_delete)
+        end
+        if File.directory?(origin)
```

Comment:
@apojomovsky I think it'd make to sense to `raise` an exception in case of misconfiguration.

Reply:
sound good, done!